services/core-web/src/components/mine/Permit/PermitConditions.tsx

```diff
@@ -103,8 +103,15 @@ const PermitConditions: FC<PermitConditionProps> = ({
     getMineReportPermitRequirementsByAmendment(permitGuid, currentAmendment?.permit_amendment_guid)
   );
 
-  const reviewAssignments = useAppSelector(getReviewAssignmentsByAmendment(currentAmendment.permit_amendment_id));
-  const userReviewAssignments = useAppSelector(getUserReviewAssignmentsByAmendment(currentAmendment.permit_amendment_id));
+  const reviewAssignments = useAppSelector(
+    getReviewAssignmentsByAmendment(currentAmendment.permit_amendment_id)
+  );
+  const userReviewAssignments = useAppSelector(
+    useMemo(
+      () => getUserReviewAssignmentsByAmendment(currentAmendment.permit_amendment_id),
+      [currentAmendment.permit_amendment_id]
```

Comment:
What happens when you assign yourself to review the conditions? Do you have to refresh the page?

Reply:
Nope.  It still updates as expected, and you can immediately edit if you assign yourself.